### ISSUE TITLE
Username should not be case sensitive

### ISSUE THREAD

**Issue by zafartahirov** [karpathy/arxiv-sanity-preserver#85]
If you create a user, say `User` and another one `useR`, and the third one `uSEr`, the website treats them as separate accounts. The behavior is sometimes confusing, and most of the time considered to be a bug (somewhat related: RFC 5321, section 2.3.11).

**Comment by kirubarajan** [comment]
I'm considering opening a pull request to fix this issue, but what would be done about prior accounts made with duplicate usernames? The options are letting them remain or programmatically altering usernames created after the original (eg. appending digits).

**Comment by zafartahirov** [comment]
I don't think this is a simple matter -- how many users are affected? If the usernames are changed, how r u going to tell users that their usernames have changed? Some users might have several accounts, because registration and login is currently the same button -- that happened with me, I accidentally typed my username in all caps, and a new user was created. Is it possible to merge those accounts?

As I said, changing the usernames would not be straightforward. The easiest one is to let the old users have case-sensitive usernames, but prevent the new ones from doing so.

**Comment by kirubarajan** [comment]
You're right, there are a lot of ways to go about this that would require the input of Andrej. I'd be happy to implement sensitive usernames and server-side verification in the meantime.

**Comment by jackvial** [comment]
A possible solution could be to gradually migrate over to using email as the account identifier. New users would be required to enter their email address and would be sent a verification email, existing users could be prompt on the site that they need to link an email address to their account.
